### PR TITLE
Events Discovery to Dev

### PR DESCRIPTION
**Pull request by ericbakenhus**
Moving all events discovery work to dev.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "gatsby build",
+    "build": "gatsby clean && gatsby build",
```
[the remark this answers is not in the document]
Comment (ericbakenhus) [reply package.json]: Just Gatsby things. 😐

[the remark this answers is not in the document]
Comment (ericbakenhus) [reply package.json]: Apparently, reading the error explains the error. Seems fine now. New optional package added.

File: package.json [hunk 2]
```diff
@@ -151,5 +151,8 @@
     "react-server-dom-webpack@0.0.0-experimental-c8b778b7f-20220825": {
       "react": "^18.0.0"
     }
+  },
+  "optionalDependencies": {
+    "@parcel/watcher-linux-x64-glibc": "^2.4.2-alpha.0"
```
Comment (sherakama) [inline package.json:156]: Optional package not optional?

Comment (sherakama) [reply package.json:156]: Just joking around. Thanks for reading and helping fix this long time bug.

Comment (ericbakenhus) [reply package.json:156]: I guess not. 🤷

**Review by sherakama — APPROVED** [review]
Fire when ready.

**Comment by sherakama** [comment]
Screenshot 2024-07-31 at 16 23 17

🎉 !

**Pull request merged**